crates/settings/src/deployment.rs

```diff
@@ -30,6 +30,28 @@ pub struct Deployment {
 #[cfg(target_family = "wasm")]
 impl_all_wasm_traits!(Deployment);
 
+impl Deployment {
+    pub fn parse_order_key(
+        documents: Vec<Arc<RwLock<StrictYaml>>>,
+        deployment_key: &str,
+    ) -> Result<String, YamlError> {
+        for document in &documents {
+            let document_read = document.read().map_err(|_| YamlError::ReadLockError)?;
+
+            if let Ok(deployments_hash) = require_hash(&document_read, Some("deployments"), None) {
+                if let Some(deployment_yaml) =
+                    deployments_hash.get(&StrictYaml::String(deployment_key.to_string()))
+                {
+                    return Ok(require_string(deployment_yaml, Some("order"), None)?);
+                }
+            }
+        }
+        Err(YamlError::ParseError(format!(
+            "order key for deployment: {deployment_key}"
```

Comment:
should this be "order key not found for deployment" ?

Reply:
yeah missed that, will fix